assembl/static/js/app/utils/tourManager.js

```diff
@@ -27,6 +27,9 @@ var TourManager = Marionette.Object.extend({
       },
 
   initialize: function() {
+    if (Ctx.isAdminApp()) {
+      return;
+    }
```

Comment:
Would it be possible to not load the tours on admin app, instead of instanciating the TourManager and hardcoding a break inside it? (creates a coupling problem?)